package.json

```diff
@@ -21,21 +21,23 @@
   },
   "dependencies": {
     "@kth/reqvars": "^2.0.1",
-    "bunyan": "^1.8.12",
+    "bluebird": "^3.7.2",
```

Comment:
Bluebird! It's been a while since we had a reason for including that dependency, good thing you removed it!